snap/hooks/install

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+mkdir -p $SNAP_COMMON/libvirt
+cp -r $SNAP/var/snap/multipass/common/libvirt/libvirtd.conf $SNAP_COMMON/libvirt
+sed -i 's/unix_sock_group = \"libvirtd\"/unix_sock_group = \"sudo\"/' $SNAP_COMMON/libvirt/libvirtd.conf
```

Comment:
Should be no need to escape the double quotes?

Will we have to rely on `sudo` membership long-term?

Reply:
I thought I ended up having to use the double quotes, but maybe that's not necessary.  It works with it, but I can try it without the escaped double quotes.

Regarding `sudo`, we need to put in some group here that will common on all distros we want to support.  We have to use system groups here since, and correct me if I'm wrong, snaps don't have a concept of separate groups.  That said, I noticed that on xenial, it uses `libvirtd` as the group, but on artful, it uses `libvirt`, so I had to change it to something both would support out of the box.  Also, Solus seems to be an issue wrt to groups, so I thought I would go with the same group we use for the multipass socket.

Reply:
Have a look at how the `lxd` snap solved this? They don't require group membership.

Reply:
This is needed for `multipass.virsh` to connect to the libvirt socket only.